src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -82,6 +82,12 @@
   <module name="org.openhab.tools.analysis.checkstyle.ImportExportedPackagesCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.ManifestPackageVersionCheck">
+     <property name="severity" value="warning" />
+     <property name="ignoreImportedPackages" value="org.junit.*,
```

Comment:
ah, sorry, and the exclude is not only for test fragments, but for all bundles - yes, then I agree that there must not be any version constraint on org.slf4j.